tests/testAutopas/tests/iterators/RegionParticleIteratorTest.cpp

```diff
@@ -14,9 +14,22 @@
 extern template class autopas::AutoPas<Molecule>;
 extern template bool autopas::AutoPas<Molecule>::iteratePairwise(EmptyFunctor<Molecule> *);
 
+using ::testing::Combine;
+using ::testing::UnorderedElementsAreArray;
+using ::testing::Values;
+using ::testing::ValuesIn;
+
+// Helper struct to represent region iterator boxes.
+struct Box {
+  std::array<double, 3> min;
+  std::array<double, 3> max;
+};
```

Comment:
I feel like we should add helper types like Point and Box to some utils class... but not in this PR

Reply:
-> #844